crypto/s2n_sequence.c

```diff
@@ -38,3 +40,17 @@ int s2n_increment_sequence_number(struct s2n_blob *sequence_number)
 
     return 0;
 }
+
+int s2n_convert_sequence_number(struct s2n_blob *sequence_number, uint64_t *output)
```

Comment:
perhaps `s2n_sequence_number_to_uint64()` would be clearer?

Reply:
Yeah that's a good point